### ISSUE TITLE
Unachievable Achievement

### ISSUE THREAD

**Issue by Ogini** [Darkosto/SkyFactory4Issues#597]
## Context

I'm hesitating to submit this, as it's possible I haven't found the workaround yet, and that there may be no clear solution.

To the matter at hand: the achievement "**Dusk Till Dawn**" requires acquiring "**Twilight Matter**", from killing Twilight Forest Mobs using _Deep Mob Learning_. But, as the Twilight Forest is prestige locked, I cannot find a way to achieve this achievement, and thus I cannot complete the page for its prestige point.

I understand there would be confusion if this achievement was hidden. The user then completes the page (Age of Power), then chooses to unlock the Twilight Forest. What would be the result? Show the achievement? As incomplete, on a completed page?

Though I must say you've probably thought more about this than I have!

**Comment by sk2048** [comment]
Thank you for reporting this, that advancement should be hidden for players without the Twilight Forest prestige, and it should not be a requirement for the page completion.

This happens in various other places. Age of Farming - if you unlock the mulch, the advancements for that show though they are not a requirement for page completion. The same for the Twilight Forest advancement in Age of Exploration.

**Comment by SpeedSlider** [comment]
"I like big box and I cannot lie" achievement appears unachievable in storage tab. I created the box builder and placed the items in the slots and get a ding but no box. We are playing on the AllTheMods Patreon Feature Pack 1 server which I don't believe is using any server software like sponge or anything. I have op role and searched through the creative menu and could only find the box builder, a backpack (white) and two drums associated with the mod. It appears that the mod is not complete or is missing items. As far as I could confirm ATM servers don't use plugins on the Patreon servers.

I did try this in a single player test world with cheats and was able to craft the item in survival mode. I also found that the default hue was 180 while in the multiple server hue 180 could not be selected it skipped around from 179 to 181. Not sure what is going on.
